### PR TITLE
truncate output.

### LINKED ISSUE #990: Internal eval logs an exception if the hastebin payload is too long
Sentry Issue: [SIR-LANCEBOT-7F](https://sentry.io/organizations/python-discord/issues/2885555776/?referrer=github_integration)

Relevant lines: https://github.com/python-discord/sir-lancebot/blob/ec6eb2e3ce4d2f0a4c3917f9eff00266a6ecf894/bot/exts/core/internal_eval/_internal_eval.py#L96-L98

Our paste service currently has a length limit of 100000. The payload should be truncated before sending it (though probably to something more reasonable than 100000). You can use the other bot's interval eval implementation as reference.

```
ClientResponseError: 400, message='Bad Request', url=URL('https://paste.pythondiscord.com/documents')
  File "bot/exts/core/internal_eval/_internal_eval.py", line 89, in _upload_output
    async with self.bot.http_session.post(
  File "aiohttp/client.py", line 1117, in __aenter__
    self._resp = await self._coro
  File "aiohttp/client.py", line 625, in _request
    resp.raise_for_status()
  File "aiohttp/client_reqrep.py", line 1000, in raise_for_status
    raise ClientResponseError(

Failed to upload `internal eval` output to paste service!
```

### PR DESCRIPTION
**Pull request by MaskDuck**
## Relevant Issues

Close #990 

## Description
I have truncated the output using the shorten_output method, limit the output to be 99980* character.
(*) the reason why is it 99980 is there should be 20 charater in the end stating that the output was truncated, they are `\n[output truncated]`

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/exts/core/internal_eval/_internal_eval.py [hunk 1]
```diff
@@ -85,9 +85,10 @@ def shorten_output(
 
     async def _upload_output(self, output: str) -> Optional[str]:
         """Upload `internal eval` output to our pastebin and return the url."""
+        data = self.shorten_output(output, max_length=99980)
```
Comment (Akarys42) [inline bot/exts/core/internal_eval/_internal_eval.py:88]: `99980` should probably be a constant

**Review by Akarys42 — CHANGES_REQUESTED** [review]
Just a tiny thing

File: bot/exts/core/internal_eval/_internal_eval.py (hunk 1 above)
Comment (MaskDuck) [reply bot/exts/core/internal_eval/_internal_eval.py:88]: it shouldn't be change?

**Comment by MaskDuck** [comment]
i mean constant lol

**Review by Akarys42 — APPROVED** [review]

**Review by brad90four — COMMENTED** [review]
Could this be done instead by setting the default `max_length` value to 99980?

```py
    @staticmethod
    def shorten_output(
            output: str,
            max_length: int = 99980,
            placeholder: str = "\n[output truncated]"
    ) -> str:
```

**Review by brad90four — APPROVED** [review]
~~LGTM, nice and clean 👌 ~~
My suggestion was a bad one, changes will be reverted. [Conversation in dev-contrib](https://discord.com/channels/267624335836053506/635950537262759947/948766129076195338)

**Comment by MaskDuck** [comment]
@brad90four reverted =)

**Comment by MaskDuck** [comment]
ok the linter doesnt complain about anything ig

**Review by brad90four — APPROVED** [review]
This definitely works

**Review by jb3 — APPROVED** [review]

**Pull request merged**